src/transformStateWithClones.js

```diff
@@ -6,8 +6,34 @@
  *
  * @return {Object[]}
  */
+
 function transformStateWithClones(state, actions) {
-  // write code here
+  const stateVers = [state];
```

Comment:
So, as I understand, at first you add `state` in array and after that you remove it from array when make return `stateVers.slice(1)`.

It's seems not good. I can suggest you make sth like that:
```suggestion
  const result = [];
  const stateCopy = { ...state };
```

When you will push your data in result and have not modified state.